src/Uno.UI.Composition/Composition/CompositionObject.cs

```diff
@@ -22,19 +22,33 @@ internal CompositionObject()
 		{
 			ApiInformation.TryRaiseNotImplemented(GetType().FullName!, "The compositor constructor is not available, as the type is not implemented");
 			Compositor = new Compositor();
+			Properties = GetProperties();
 		}
 
 		internal CompositionObject(Compositor compositor)
 		{
 			Compositor = compositor;
+			Properties = GetProperties();
```

Comment:
Could we lazy init this ? We are creating a lot of `CompositionObject`, it will probably be a win to create it only when needed.

Reply:
Sure.